examples/jsm/loaders/LDrawLoader.js

```diff
@@ -1949,15 +1949,29 @@ class LDrawLoader extends Loader {
 	}
 
 	setMaterials( materials ) {
+		this.clearMaterials();
+		this.addMaterials( materials );
```

Comment:
nit: lets adhere to the code style (blank lines to pad before / after function braces), and return `this` from the new functions so we're consistent with the existing function behavior.

Reply:
Done! I was fighting against my own linter a bit :)